components/json_page/index.coffee

```diff
@@ -30,13 +30,13 @@ module.exports = class JSONPage
         .end (err, res) =>
```

Comment:
I think this code wasn't updated when superagent changed its API to use the node-style error-first argument. We should handle `err` here instead.

Or bonus points if you drop the `.end` all-together and use superagent's new promise-based API.